mods/ra/rules/aircraft.yaml

```diff
@@ -33,6 +33,8 @@ BADR:
 		Interval: 2
 	-EjectOnDeath:
 	RejectsOrders:
+    GivesExperience:
```

Comment:
A lint test should ignore the LongDesc, as d2k has a few with spaces, not tabs.

But that might be another ticket.